Documentation/ApiOverview/Events/Events/FrontendLogin/_LogoutConfirmedEvent/_Services.yaml

```diff
@@ -0,0 +1,13 @@
+services:
```

Comment:
I dont think it is good to omit that part. ppl will just copy the code and end up with something not working. And even I myself always have to go and look it up. If there is ever a change to the suggested format it will be an easy search and replace

Reply:
That's why there is a link to the full example after the snippet - also with exclude and that stuff (which is necessary). We agreed on that some months ago.

Reply:
Its just like with the "final" in each and every example class. ppl will copy this stuff without much thinking.

Reply:
I can live with that, but the examples sections of all events should be build the same. I invested a lot of time to clean up and synchronise the examples structure in the event chapter. So, if I approve this, then please, also adjust all the other examples in the same way.

For now, as I already suggested: This PR should be backported to 12.4, ideally without separate changes in that PR for 12.4 branch. So, I propose to use a Services.yaml like in 12.4 (without using an attribute), and like in the other examples, and in a follow-up adjust it for v13.

Changes in the Services.yaml for all events should be discussed separately (for all events) and not for this one,

Reply:
I changed it to your way for now. However it will be very hard or impossible to encourage user contribution and still apply such strict rules. Also its not really like anyone would go from one example to the next and compare if they are all the same.